pool/web/handler/admin_settings_actor_permissions.ml

```diff
@@ -0,0 +1,207 @@
+open CCFun.Infix
+open Utils.Lwt_result.Infix
+module HttpUtils = Http_utils
+module Message = HttpUtils.Message
+module Field = Pool_common.Message.Field
+
+let src = Logs.Src.create "handler.admin.settings_actor_permissions"
+let active_navigation = "/admin/settings/actor-permission"
+
+let show req =
+  HttpUtils.Htmx.handler
+    ~active_navigation
+    ~error_path:"/"
+    ~query:(module Guard.ActorPermission)
+    ~create_layout:General.create_tenant_layout
+    req
+  @@ fun ({ Pool_context.database_label; _ } as context) query ->
+  let%lwt permissions, query =
+    Guard.Persistence.ActorPermission.find_by query database_label
+  in
+  let open Page.Admin.Settings.ActorPermission in
+  (if HttpUtils.Htmx.is_hx_request req then list else index)
+    context
+    permissions
+    query
+  |> Lwt_result.return
+;;
+
+let delete req =
+  let result { Pool_context.database_label; _ } =
+    let tags = Pool_context.Logger.Tags.req req in
+    let permission =
+      Sihl.Web.Request.to_urlencoded req
+      ||> HttpUtils.find_in_urlencoded Field.Permission
+      >== fun permission ->
+      let read = Yojson.Safe.from_string %> Guard.ActorPermission.of_yojson in
+      CCResult.map_err
+        Pool_common.Message.authorization
+        (try read permission with
+         | _ -> Error "Undefined Yojson for actor permission.")
+    in
+    let events =
+      Cqrs_command.Guardian_command.DeleteActorPermission.handle ~tags
+    in
+    let handle = function
```

Comment:
Do you hide the error on purpose?

Reply:
yes, here the error for the user is only "Permission denied"